recipes/packit/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.24" %}
+{% set hash_value = "f22cbf399a5fcded061c566691be58d421b6c2b668ee0602665749a0266715f7" %}
+
+package:
+  name: python-packit
+  version: '{{ version }}'
+
+source:
+  url: https://files.pythonhosted.org/packages/58/4f/0ac86eb3cc9ce74f87de314faf33426798052b5b17a8e18bb4f0a08686b4/packit-0.24.tar.gz
+  sha256: '{{ hash_value }}'
+  patches:
+    - version.patch
+
+build:
+  skip: true  # [win]
```

Comment:
I think this recipe can be ``noarch: python`` right?
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds

Reply:
You're right